indigo_api/models/documents.py

```diff
@@ -544,6 +547,20 @@ def change_date(self, new_date, user, comment=None):
             self.expression_date = new_date
             self.save_with_revision(user, comment=comment)
 
+    def update_provision_xml(self, provision_eid, new_provision_eid, provision_xml):
+        generator = XmlGenerator(self.frbr_uri)
+        xml = etree.fromstring(provision_xml)
+        akn_provision = generator.wrap_akn(xml)
+        portion = Portion(etree.tostring(akn_provision, encoding='unicode'))
+        updated_provision = portion.get_portion_element(new_provision_eid)
```

Comment:
seems odd to have to have the new_provision_id, can't we just use the portionBody first child element?